portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,11 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	document.getElementById("inner-content").style.display = "block";
+	const response = await fetch('/data');
+	const hello = await response.text();
+	document.getElementById("my-info").innerText = hello;
+	console.log(`Fetch Hello World!`);
```

Comment:
uber nit - i don't know this for sure, but based on the style guide for quotes I think we should be using single quotes where possible and template strings when the string contains a single quote.

Reply:
thanks! will do!